SoftLayer/CLI/image/list.py

```diff
@@ -29,11 +29,18 @@ def cli(env, name, public):
         for image in image_mgr.list_public_images(name=name, mask=image_mod.MASK):
             images.append(image)
 
-    table = formatting.Table(['id', 'name', 'type', 'visibility', 'account'])
+    table = formatting.Table(['id', 'name', 'type', 'visibility', 'account', 'os', 'created', 'notes'])
```

Comment:
please change to capital letter 
```suggestion
    table = formatting.Table(['Id', 'Name', 'Type', 'Visibility', 'Account', 'OS', 'Created', 'Notes'])
```

Reply:
Done